cassandra/Dockerfile

```diff
@@ -1,22 +1,30 @@
-# Share the same base image to reduce layers used in testing
-FROM openzipkin/jre-full:1.8.0_191
-MAINTAINER OpenZipkin "http://zipkin.io/"
+FROM openjdk:8-jre-alpine
```

Comment:
did you want to use the base jre alpine here?

Reply:
Yeah - I need an image with java, curl and stuff to run Cassandra while the setup process populates its data partition with the zipkin schema. We just copy it all out in the second stage of the build so it doesn't matter so much what this base is for the builder.

Reply:
ack